### DISCUSSION TITLE
MPM application parameters

### DISCUSSION THREAD

**Discussion by MiguelARV16** [KratosMultiphysics/Kratos#12906] [Q&A]
Hello.
I am working with Kratos, specifically with the MPM application. I have made several tests on the models I am interested in, modifying some parameters associated to the real materials; however, the modeling is not completed and the following message appears during the calculation process.

![input_matrix](https://github.com/user-attachments/assets/e678f3dd-9487-4e0a-a64e-e6e9bba40f7a)

Please, Could you help me with any idea of what I could consider or modify?

**Comment by AlejandroCornejo** [comment]
Can you put the echo level to 1 to see the iterative process information? This print usually arises when divergence occurs and the Jacobian of an element becomes negative... @KratosMultiphysics/mpm

**Reply by MiguelARV16** [reply]
this is part of the calculation process, now with 1 on the echo level 
![input_matrix_echo_1](https://github.com/user-attachments/assets/3edd1bfb-9012-4a45-bb3c-88ece25775ca)
